ProfFile.hs

```diff
@@ -63,40 +63,71 @@ parseLine format s =
         _ -> Left $ "Malformed .prof file line:\n" ++ s
     IncludesSources ->
       case words s of
-        (costCentre:module_:_:no:entries:indTime:indAlloc:inhTime:inhAlloc:other) ->
-          parse' costCentre module_ no entries indTime indAlloc inhTime inhAlloc other
+        (costCentre:module_:rest) | (no:entries:indTime:indAlloc:inhTime:inhAlloc:other) <- dropSRC rest ->
```

Comment:
The column is present since GHC 8.0.2. @cocreature added support for it in #4 and already took care of backwards compatibility (the `ProfFormat` parameter of `parseLine` indicates which format is in use).